src/terrain/routes/bags.clj

```diff
@@ -0,0 +1,59 @@
+(ns terrain.routes.bags
+  (:use [common-swagger-api.schema]
+        [ring.util.http-response :only [ok]]
+        [terrain.routes.schemas.bags]
+        [terrain.auth.user-attributes :only [current-user]]
+        [terrain.services.bags]
+        [terrain.util :only [optional-routes]])
+  (:require [terrain.util.config :as config]
+            [clojure.tools.logging :as log]))
+
+(defn bag-routes
+  []
+  (optional-routes
+   [config/bag-routes-enabled]
+
+   (context "/bags" []
+     :tags ["bags"]
+
+     (HEAD "/" []
+       :summary     HasBagsSummary
+       :description HasBagsDescription
+       (ok (has-bags (:username current-user))))
```

Comment:
In cases where you're not returning a response body, it's probably better to move the endpoint body out of the call to the function generating the HTTP response map, for example:

``` clojure
     (HEAD "/" []
       :summary     HasBagsSummary
       :description HasBagsDescription
       (has-bags (:username current-user))
       (ok))
```

Doing this helps to ensure that you don't get any unintended cruft in the response body.

Reply:
This should be fixed now.